opentelemetry-api/src/opentelemetry/propagate/__init__.py

```diff
@@ -129,6 +129,10 @@ def inject(
     "tracecontext,baggage",
 )
 
+if environ_propagators.lower() == "none":
+    propagators = []
+else:
+    propagators = environ_propagators.split(",")
 
 for propagator in environ_propagators.split(","):
```

Comment:
Even if environ_propagators is set to "none", the for-loop after the if-else block still runs. I think the idea is to run only if not none no?

Reply:
@emdneto You're right! The for loop shouldn’t run if environ_propagators is set to "none", as we don’t want to load any propagators in that case. I'll move the for loop inside the else block.